principia_make.sh

```diff
@@ -0,0 +1,14 @@
+PRINCIPIA_CXX_FLAGS=-std=c++14
+PRINCIPIA_MACOS_C_FLAGS=
+
+principia_configure
+
```

Comment:
Actually I would like to separate this file in two bits at this line: everything below should be executed in the pipeline as part of the Make task group with variable overrides.  Everything above effectively simulates what the pipeline does.  (And some if-this-then-that needs to happen.)

Reply:
These flags are only used within the make task, so we should try to override them in one place and not three.

Perhaps they should be removed from the variable group, but that is a different question.